packages/@react-spectrum/menu/docs/ActionMenu.mdx

```diff
@@ -0,0 +1,259 @@
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/popover/'}
+  ]} />
+
+## Example
+
+```tsx example
+<ActionMenu>
+  <Item>Cut</Item>
+  <Item>Copy</Item>
+  <Item>Paste</Item>
+</ActionMenu>
+```
+
+## Content
+
+ActionMenu follows the [Collection Components](../react-stately/collections.html) API,
+accepting both static and dynamic collections. See [Menu](Menu.html#content) for further explanation.
+
+```tsx example
+function Example() {
+  let actionMenuItems = [
+    {name: 'Cut'},
+    {name: 'Copy'},
+    {name: 'Paste'},
+    {name: 'Replace'}
+  ];
+
+  return (
+    <ActionMenu items={actionMenuItems}>
+      {item => <Item key={item.name}>{item.name}</Item>}
+    </ActionMenu>
+  );
+}
+```
+
+### Trays
+On mobile, ActionMenus automatically display in a tray instead of a popover to improve usability.
+
+### Internationalization
+In order to internationalize an ActionMenu, the content provided to all child items should be localized.
+For languages that are read right-to-left (e.g. Hebrew and Arabic), the layout of the ActionMenu is flipped.
+
+## Events
+ActionMenu supports the `onAction` callback which receives the selected item. See [Selection](#selection) for more information.
+
+```tsx example
+function Example() {
+  let [action, setAction] = React.useState('');
+
+  return (
+    <>
+      <ActionMenu onAction={setAction}>
+        <Item key="cut">Cut</Item>
+        <Item key="copy">Copy</Item>
+        <Item key="paste">Paste</Item>
+      </ActionMenu>
+      <p>Action: {action}</p>
+    </>
+  );
+}
+```
+
+## Sections
+ActionMenu supports sections in order to group options. Sections can be used by wrapping groups of Items in a `Section` component. Each `Section` takes a `title` and `key` prop.
+
+### Static Items
+
+```tsx example
+<ActionMenu>
+  <Section title="Styles">
+    <Item key="bold">Bold</Item>
+    <Item key="underline">Underline</Item>
+  </Section>
+  <Section title="Align">
+    <Item key="left">Left</Item>
+    <Item key="middle">Middle</Item>
+    <Item key="right">Right</Item>
+  </Section>
+</ActionMenu>
+```
+
+### Dynamic Items
+
+Sections used with dynamic items are populated from a hierarchical data structure. Similarly to the props on ActionMenu, `<Section>` takes an array of data using the `items` prop.
+
+```tsx example
+function Example() {
+  let openWindows = [
+    {
+      name: 'Left Panel',
+      id: 'left',
+      children: [
+        {id: 1, name: 'Final Copy (1)'}
+      ]
+    },
+    {
+      name: 'Right Panel',
+      id: 'right',
+      children: [
+        {id: 2, name: 'index.ts'},
+        {id: 3, name: 'package.json'},
+        {id: 4, name: 'license.txt'}
+      ]
+    }
+  ];
+
+  return (
+    <ActionMenu
+      items={openWindows}>
+      {item => (
+        <Section items={item.children} title={item.name}>
+          {item => <Item>{item.name}</Item>}
+        </Section>
+      )}
+    </ActionMenu>
+  );
+}
+```
```

Comment:
Can we make the items in the above examples actions as this component would be used rather than items you would select? Cut/copy/paste as used in the other examples are good.